deploy/grafana-dashboard/general_row.py

```diff
@@ -0,0 +1,112 @@
+from grafanalib.core import RowPanel, GridPos, Stat, TimeSeries, Heatmap, BarGauge, GAUGE_DISPLAY_MODE_GRADIENT
+from grafanalib.influxdb import InfluxDBTarget
+
+from color_mapping import grey_all_mapping
+from util import read_query
+
+requests_per_minute = TimeSeries(
+    title="Requests per minute",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("requests-per-minute"))],
+    gridPos=GridPos(h=9, w=8, x=0, y=1),
+    scaleDistributionType="log",
+    extraJson=grey_all_mapping
+)
+
+request_latency = Heatmap(
+    title="Request Latency",
```

Comment:
I cannot tell why, but there seems to be a difference in the heat map (maybe due to the resolution and number of data points)?

Grafana as code:

![Bildschirm­foto 2022-10-27 um 00 01 29](https://user-images.githubusercontent.com/7300329/198148119-bf032789-22aa-4834-9e1d-be456b241896.png)

Current dashboard:

![Bildschirm­foto 2022-10-27 um 00 01 33](https://user-images.githubusercontent.com/7300329/198148135-0ed6043e-13ce-49f1-b102-99fc359aad8e.png)

Reply:
Yeah, it is because of the static number of data points. I have now changed it to a dynamic number of data points (as it was originally).